src/random_read_worker.rs

```diff
@@ -94,7 +98,9 @@ impl RandomReadWorker {
 
         success?;
 
-        let end_time = Instant::now();
-        Ok((query_times, end_time.duration_since(start_time)))
+        Ok(query_qps
```

Comment:
what's the formula for qps? adding it as a comment would be helpful

Reply:
added comment